packages/admin-ui/src/combobox/combobox.style.ts

```diff
@@ -1,4 +1,5 @@
-import { style, styleVariants } from '@vtex/admin-ui-core'
+import { csx, cx, dataAttr, style, styleVariants } from '@vtex/admin-ui-core'
+import { action as actionColorScheme } from '../button/button.css'
```

Comment:
It looks to me that the "actionSchema" is used in several places, does it make sense to be something from the button itself, or should it be a utility?

IMO we can let it be attached to the button for now, but once our design language gets more mature it will probably make sense to turn it into something bigger. Saying that I would export it as `actionSchema`, what do you think?
```suggestion
import { actionSchema } from '../button/button.css'
```

Reply:
Agreed. I think the schema should probably be reviewed too

Reply:
Do we have an issue to keep track of this future change?